From 7e039c4ff7466d253ac8a9511ffbe02b9ffc721c Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Sun, 15 Sep 2019 13:47:19 +0200
Subject: [PATCH] meson: Define TUNNEL_SINK for module-tunnel-sink

This was missing, so it was building another tunnel-source instead.

Fixes https://bugs.archlinux.org/task/63755
---
 src/modules/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/meson.build b/src/modules/meson.build
index 92d5871f9..dd2241f42 100644
--- a/src/modules/meson.build
+++ b/src/modules/meson.build
@@ -55,7 +55,7 @@ all_modules = [
   [ 'module-suspend-on-idle', 'module-suspend-on-idle.c' ],
   [ 'module-switch-on-connect', 'module-switch-on-connect.c' ],
   [ 'module-switch-on-port-available', 'module-switch-on-port-available.c' ],
-  [ 'module-tunnel-sink', 'module-tunnel.c', [], [], [x11_dep] ],
+  [ 'module-tunnel-sink', 'module-tunnel.c', [], ['-DTUNNEL_SINK=1'], [x11_dep] ],
   [ 'module-tunnel-sink-new', 'module-tunnel-sink-new.c' ],
   [ 'module-tunnel-source', 'module-tunnel.c', [], [], [x11_dep] ],
   [ 'module-tunnel-source-new', 'module-tunnel-source-new.c' ],
-- 
2.24.1

